### PR TITLE
kb(raddataform): bind toggle button in editform

### PR DESCRIPTION
**Pull request by edrohler**
Client wanted to bind the toggle of a RadButton in RadDataForm in

### CODE REVIEW

File: knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+position: 
+tags: RadDataForm, EditTemplate, RadButton, ButtonToggle
+ticketid: 1441058
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
```
Comment (PMcDonough11) [inline knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md]: Add Product Version (like in [this KB](https://docs.telerik.com/devtools/aspnet-ajax/knowledge-base/gantt-custom-hyperlink-column))

File: knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+## Solution
+In the below sample, the IsSomething value is either a 0 or 1, i.e. true or false. In the [RadDataForm ItemTemplate](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/data-binding/declarative-data-source) binding can use the [DataBinder.Eval Method](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.databinder.eval). This doesn't work when Editing. In this case, set the [DataKeyNames](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/dataform-object) and use the [OnItemCreated](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/events) Event of the RadDataForm to set the Toggle State using the SetSelectedToggleByText or SetSelectedToggleByValue.
```
Comment (PMcDonough11) [inline knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md]: Change OnItemCreated to [ItemCreated ](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/events)

File: knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+}
+```
+## See Also
+
+*   [Toggle Type Telerik RadButton](https://docs.telerik.com/devtools/aspnet-ajax/controls/button/button-types/toggle-button)
+*   [Editing in a RadDataForm](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/data-editing/manual-crud-operations)
+*   [RadDataForm ItemTemplate](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/data-binding/declarative-data-source)
+*   [DataBinder.Eval Method](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.databinder.eval?view=netframework-4.8)
+*   [DataKeyNames](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/dataform-object)
+*   [OnItemCreated](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/events)
```
Comment (PMcDonough11) [inline knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md]: Change to [ItemCreated](https://docs.telerik.com/devtools/aspnet-ajax/controls/dataform/server-side-programming/events)

File: knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md (hunk 1 above)
Comment (edrohler) [reply knowledge-base/raddataform-bind-toggle-button-in-a-edittemplate.md]: Done.

**Pull request merged**